data/campaigns/Under_the_Burning_Suns/utils/abilities.cfg

```diff
@@ -1926,20 +1944,5 @@ _ "After using this attack, you can’t use it during your next two turns."#endd
 
 [resource]
     id=utbs_ability_events
-    {UTBS_TAILWIND_EVENTS}
-    {WEAPON_SPECIAL_RAY_EVENTS}
-    {WEAPON_SPECIAL_PIERCE_EVENTS}
-    {WEAPON_SPECIAL_PUNCTURE_EVENTS}
-    {ABILITY_OPPORTUNIST_EVENTS}
-    {WEAPON_SPECIAL_NOVA_EVENTS}
-    {WEAPON_SPECIAL_USES_ATTACKS_EVENTS}
-    {WEAPON_SPECIAL_ONCE_PER_TURN_EVENTS}
-    {WEAPON_SPECIAL_RAIN_OF_ARROWS_EVENTS}
-    {WEAPON_SPECIAL_SWORD_DANCE_EVENTS}
-    {WEAPON_SPECIAL_COOLDOWN_EVENTS 2}
-    {WEAPON_SPECIAL_COOLDOWN_EVENTS 3}
-    {WEAPON_SPECIAL_TAUNT_EVENTS}
-    {SONG_VERSE_EVENTS 2}
-    {SONG_VERSE_EVENTS 3}
     {SONG_VERSE_ILLUMINATES}
```

Comment:
Why not move this in the related ability as well?

Reply:
it's done now